README.md

```diff
@@ -1,4 +1,9 @@
-Travis-CI: [![Build Status](https://travis-ci.org/ipa320/cob_android.svg?branch=indigo_dev)](https://travis-ci.org/ipa320/cob_android)
-
 cob_android
 ===========
+
+Travis-CI: [![Build Status](https://travis-ci.org/ipa320/cob_android.svg?branch=indigo_dev)](https://travis-ci.org/ipa320/cob_android)
+
+ROS-Distro-Support:
+ - ROS Indigo Igloo (branch: `indigo_dev`)
```

Comment:
We could link to the ROS buildfarm devel jobs for (additional) distro-specific status icon